### DISCUSSION TITLE
Stuck on "Rebuilding page tree: [ COMPLETED ]" on npm run start

### DISCUSSION THREAD

**Discussion by Dolyyyy** [requarks/wiki#6640] [Help / Questions] [answered]
OS: Ubuntu 22.04 without Docker
Wiki.js version: 2.5.295
Database engine: Maria DB 10.11.4

I've this problem :
![image](https://github.com/requarks/wiki/assets/66006040/8af4a510-91b2-4e04-a785-1508762a343e)

I try to run WikiJS without Docker, i've this problem.

**Comment by fnafox** [comment, marked as answer]
I see no problems here, the software is running as intended. Navigate to `localhost:5355` to use the wiki.

**Comment by Dolyyyy** [comment]
> I see no problems here, the software is running as intended. Navigate to `localhost:5355` to use the wiki.

That's strange because i've nothing on the port, maybe i have to change some firewall rules. Thank's for you help !
